test/suites/clustering.sh

```diff
@@ -2333,4 +2359,7 @@ test_clustering_image_refresh() {
   kill_lxd "${LXD_TWO_DIR}"
   kill_lxd "${LXD_THREE_DIR}"
   kill_lxd "${LXD_REMOTE_DIR}"
+
+  # shellcheck disable=SC2034
+  LXD_NETNS=
```

Comment:
@monstermunchkin whats this for (perhaps a comment would be prudent here as it looks a little odd)

Reply:
@tomponline i386 tests had a problem since this tests sets LXD_NETNS at one stage. Unsetting it solves the issue. I have no clue why this is a problem on i386 and not on other architectures.